target/linux/ath79/mikrotik/base-files/etc/board.d/02_network

```diff
@@ -23,6 +23,20 @@ ath79_setup_interfaces()
 	mikrotik,routerboard-wapr-2nd)
 		ucidef_set_interface_lan "eth0"
 		;;
+	mikrotik,routerboard-2011il)
```

Comment:
Technically, 2 comes before 4 ...

Reply:
Right, but my reasoning was that 493 is before 2011. I can switch it around if that would be more in line with the rest of OpenWrt, though.

Reply:
At some point, one will have a program (re)sorting it, and then all attempts to sort by "numbers" are pointless and cause extra rearrangements.

Reply:
Fixed.